src/main/java/models/SendRequestMessage.java

```diff
@@ -19,4 +19,5 @@
     private String template;
     private Object content;
     private Object metadata;
+    private Object timeout;
```

Comment:
does this have to be a number or an object?

Reply:
timeout is an object. This is what we want.

```json
{
"timeout": {
    "message": 360000
  }
}
```